build.gradle

```diff
@@ -29,35 +29,36 @@ javaPlatform {
 }
 
 dependencies {
+    api platform('org.jetbrains.kotlin:kotlin-bom:1.6.21')
+    api platform("io.grpc:grpc-bom:$grpcVersion")
+    api platform('com.fasterxml.jackson:jackson-bom:2.13.3')
+
     constraints {
         // Libraries
         api("com.rabbitmq:amqp-client:5.9.0")
-        api("com.datastax.cassandra:cassandra-driver-core:3.6.0")
+        api("com.datastax.oss:java-driver-core:$cassandraDriver")
+        api("com.datastax.oss:java-driver-mapper-processor:$cassandraDriver")
+        api("com.datastax.oss:java-driver-mapper-runtime:$cassandraDriver")
+        api("com.datastax.oss:java-driver-query-builder:$cassandraDriver")
 
-        api("com.fasterxml.jackson.core:jackson-core:${jacksonVersion}")
-        api("com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}")
-        api("com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}")
-        api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}")
-        api("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jacksonVersion}")
-        api("com.fasterxml.jackson.module:jackson-module-kotlin:${jacksonVersion}")
+        api("org.jetbrains:annotations:23.0.0")
 
-        api("org.jetbrains:annotations:20.0.0")
+        api("org.apache.commons:commons-lang3:3.12.0")
+        api('commons-io:commons-io:2.11.0')
+        api("commons-cli:commons-cli:1.5.0")
+        api("org.apache.commons:commons-collections4:4.4")
 
-        api("org.apache.commons:commons-lang3:3.8.1")
-
-        api("com.google.guava:guava:29.0-jre")
+        api("com.google.guava:guava:31.1-jre")
 
         api("com.google.protobuf:protobuf-java-util:${protobufVersion}")
         api("com.google.protobuf:protoc:${protobufVersion}")
-        api("io.grpc:grpc-protobuf:${grpcVersion}")
-        api("io.grpc:grpc-stub:${grpcVersion}")
-        api("io.grpc:grpc-core:${grpcVersion}")
-        api("io.grpc:grpc-netty:${grpcVersion}")
-        api("io.grpc:protoc-gen-grpc-java:${grpcVersion}")
+
+        api ("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlinVersion}")
```

Comment:
Why do you need this? You already have **org.jetbrains.kotlin:kotlin-bom** in dependencies